swift-sdk/Constants.swift

```diff
@@ -39,6 +39,7 @@ enum Const {
         static let updateEmail = "users/updateEmail"
         static let updateSubscriptions = "users/updateSubscriptions"
         static let getRemoteConfiguration = "mobile/getRemoteConfiguration"
+        static let embeddedMessages = "flex/messages"
     }
```

Comment:
Interesting so the endpoint is "messages" instead of "getMessages". I know it is a lot of work to change it at this point.

Reply:
Yes, the function is named `getMessages`. The way I'm thinking is that there are messages at the endpoint, and the SDK is specifically getting them. It doesn't need to be changed except for the namespace.